builtin/credential/oidc/oidctesting/provider.go

```diff
@@ -0,0 +1,200 @@
+/*
```

Comment:
As noted in that PR we'd like to move it over to go-oidc without using the old sub packages. If we get a reviewer from vault I'll port it to go-oidc.